AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSection.xml

```diff
@@ -8,7 +8,7 @@
 
 <sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
     <section name="AdobeStockSection">
-        <element name="slideOutPanelButton" type="button" selector="[data-ui-id=wysiwyg-images-content-search-adobe-stock-button]"/>
+        <element name="slideOutPanelButton" type="button" selector="//*[@data-ui-id='wysiwyg-images-content-search-adobe-stock-button']"/>
```

Comment:
What is the reason for changing this selector?

Reply:
Seems like i'm did this  for mistake, i revert it back. thanks.